application/helpers/remotecontrol/remotecontrol_handle.php

```diff
@@ -2094,7 +2094,7 @@ public function delete_participants($sSessionKey, $iSurveyID, $aTokenIDs)
 
             if (Permission::model()->hasSurveyPermission($iSurveyID, 'tokens', 'delete')) {
                 if (!tableExists("{{tokens_$iSurveyID}}")) {
-                                    return array('status' => 'Error: No survey participants table');
+                                    return array('status' => 'Error: No survey participants list');
```

Comment:
Remove plural on all these texts, e.g. 'survey participant list' (not 'survey participants list')

Reply:
Applied the change